app/controllers/books_controller.rb

```diff
@@ -0,0 +1,74 @@
+# Manages the "Book" model
+class BooksController < ApplicationController
```

Comment:
I think we'll want one auth rule that applies to all admin controllers, and no auth on any of the other controller (the rest of the site is public). Hence my suggestion. But easy to change later.